app/components/primer/image.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use `Image` to render images. It can be rendered using the `primer_image` helper.
```

Comment:
I wonder if we should make this simpler and instead come up with a standard way of indicating the helper method name somewhere else?

```suggestion
  # Use `Image` to render images.
```